cordova-lib/src/cordova/plugin.js

```diff
@@ -286,7 +286,7 @@ module.exports = function plugin(command, targets, opts) {
                                     var configXml = new ConfigParser(configPath);
                                     configXml.removePlugin(target);
                                     configXml.write();
-                                    events.emit('results', 'config.xml entry for ' +target+ ' is removed');
+                                    events.emit('results', 'config.xml entry for plugin ' + target + ' was removed');
```

Comment:
What is the 'results' channel??

There is a similar log line for removing a platform from config.xml, only it comes before the change.